app/src/main/java/com/example/trainingintive/dog_images_feature/presentation/AlbumViewModel.kt

```diff
@@ -0,0 +1,56 @@
+package com.example.trainingintive.dog_images_feature.presentation
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import com.example.trainingintive.dog_images_feature.data.repository.DogImageRepository
+import com.example.trainingintive.dog_images_feature.domain.model.DogImageUrl
+import com.example.trainingintive.navigators.MainNavigator
+import com.example.trainingintive.rx.SchedulersProvider
+import com.example.trainingintive.util.ErrorMessageId
+import com.example.trainingintive.util.MainScreenEvent
+import java.lang.Exception
+import javax.inject.Inject
+
+class AlbumViewModel @Inject constructor(
+    private val repository: DogImageRepository,
+    private val schedulers: SchedulersProvider,
+    private val navigator: MainNavigator
+) : ViewModel() {
+
+    private val _imageUrls = MutableLiveData<List<DogImageUrl>>(emptyList())
+    val imageUrls: LiveData<List<DogImageUrl>> = _imageUrls
+
+    init {
+        getAllImages()
+    }
+
+    fun getAllImages() {
```

Comment:
Why it is public? It will be called outside ViewModel later? (Same for others functions)